scripts/etc/init.d/rcS

```diff
@@ -118,6 +118,30 @@ mdev -s
 
 klogd -c 1
 
+# Check if there's an alternative rcS file and excute it
+# instead of this file. Only do this if this isn't the
+# alternative script already
+if [ -z ${am_subscript} ]; then
+    mkdir -p /boot
+    mount $bootpartition /boot
+    if [ -e /boot/config/installer/rcS ]; then
+        cp /boot/config/installer/rcS /rcS
```

Comment:
Is it wise to store the file directly in the root of the filesystem (vs in a subdirectory of it)?